### ISSUE TITLE
repeated debug message showed in the log file

### ISSUE THREAD

**Issue by Tontonitch** [mickem/nscp#502]
### Issue and Steps to Reproduce

Hello,

I'm facing a recurring debug message filling the log file, even if debug is not enabled:

```
2017-11-15 20:33:46: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:33:49: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:33:53: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:14: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:22: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:24: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:24: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:38: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:41: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:47: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:49: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:34:54: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:35:15: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:35:22: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:35:24: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:35:24: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:35:38: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:35:41: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
2017-11-15 20:35:46: debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file
```

I would like to avoid that message to be generated. Is there a way to do that?

### Expected Behavior

No debug message

### Actual Behavior

The following debug message is generated:
`debug:c:\source\0.5.1\service\logger\nsclient_logger.cpp:52: Creating logger: threaded-file`

### Details

* NSClient++ version: 0.5.1.1046
* OS and Version: Windows Server 2012
* Checking from: Icinga2
* Checking with: nscp client (nscp-local checks) and check_nscp_api.exe

**Comment by Tontonitch** [comment]
For info, below the config file.
[nsclient.ini.txt](https://github.com/mickem/nscp/files/1476387/nsclient.ini.txt)

**Comment by mickem** [comment]
Fixed in 0.5.2.20

**Comment by Tontonitch** [comment]
Tested the 0.5.2.20, fix works.
Thanks
